backend/core/src/listings/dto/filter-type-to-field-map.ts

```diff
@@ -14,7 +14,7 @@ type keysWithMappedField = Exclude<
 export const filterTypeToFieldMap: Record<keysWithMappedField, string> = {
   status: "listings.status",
   name: "listings.name",
-  bedrooms: "unitTypeRef.num_bedrooms",
+  bedrooms: "summaryUnitType.num_bedrooms",
```

Comment:
Nice find! Is this a bug in bloom's repo as well or do they have a `unitTypeRef` that we don't?

Reply:
We changed the name of the SQL ref in https://github.com/CityOfDetroit/bloom/pull/538. That PR was never upstreamed to bloom, so their naming is still consistent.